pinot-common/pom.xml

```diff
@@ -297,6 +297,10 @@
       <groupId>com.linkedin.pinot</groupId>
       <artifactId>pinot-filesystem</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.media</groupId>
+      <artifactId>jersey-media-multipart</artifactId>
```

Comment:
It's for the metadata provider, which I had to put here because all the other segment metadata classes are here too. It was throwing dependency errors in core